### DISCUSSION TITLE
Book marks transfer

### DISCUSSION THREAD

**Discussion by sktanony** [dailydotdev/daily#1171] [Feature Requests]
Hey **Daily.dev community**

Hope you all doing well. I don't know if I am writing this to the community channel or the support team. The reason of this request is, I want to use daily.dev as I liked your idea of creating a personalized feed for developers and tech enthusiasts I want to use it without any inconvenience. But I didn't find any of **my bookmarks** I added on my chrome earlier after converting my chrome feed to daily.dev feed. If this feature already exists kindly guide me and if it doesn't I request you to add this feature to clone all my chrome profile bookmarks to the feed so I can easily access my saved, favourite sites. Hope you appreciate my request.

Expecting a response from you ASAP. 

**Best regards** 
Muhammad Mustafa Wafai

**Comment by nimrodkra** [comment]
We call it shortcuts, once you open a new tab you should be able to see a button “Add shortcuts”. Let me know if it worked, or send a screenshot of your new feed and we’ll try to assist. 
 
If you’re referring to the browser’s bookmarks here’s a guide to get them back to appear: https://docs.daily.dev/docs/getting-started/browser-extension-installation#my-chromes-bookmarks-bar-is-gone-how-can-i-bring-it-back

**Reply by sktanony** [reply]
yooo it worked thanks
